background/background.js

```diff
@@ -87,23 +81,23 @@ var browserCommands, contextMenus;
 // register all listeners
 msg.on(onRuntimeMessage);
 
+// tell apply.js to refresh styles for non-committed navigation
 navigatorUtil.onUrlChange(({tabId, frameId}, type) => {
-  if (type === 'committed') {
-    // styles would be updated when content script is injected.
-    return;
+  if (type !== 'committed') {
+    msg.sendTab(tabId, {method: 'urlChanged'}, {frameId})
+      .catch(msg.ignoreError);
+  }
+});
+
+tabManager.onUpdate(({tabId, url, oldUrl = ''}) => {
+  if (usercssHelper.testUrl(url) && !oldUrl.startsWith(URLS.installUsercss)) {
+    usercssHelper.testContents(tabId, url).then(data => {
+      if (data.code) usercssHelper.openInstallerPage(tabId, url, data);
```

Comment:
`data` can be `false` so we need `if (data && data.code) ...`.

Reply:
No, `false` is perfectly fine as you can see in console: `(false).constructor`.
Only `null` and `undefined` can't have properties.

Reply:
I think we have a disagreement about "perfectly fine".

Reply:
I meant there's no technical `need` for the change.